utils/wikize_refs.py

```diff
@@ -74,12 +85,33 @@ def parse_args():
     parser.add_argument("-w", "--warn",
                       default=False,
                       action="store_true",
-                      help="Warn instead of error during (most) error checks.")
+                      help="Warn instead of exit during (most) error checks.")
+
+    parser.add_argument("-c", "--check-links",
+                      type=int, default=0,
+                      help="Specify a timeout>0 in seconds for checking for broken links.")
```

Comment:
Please mention adding this option will result in  **network communication** (which one would not ordinarily expect for a simple utility)!

Reply:
Done